core/src/test/java/com/yahoo/imapnio/client/ImapClientRespReaderTest.java

```diff
@@ -0,0 +1,315 @@
+package com.yahoo.imapnio.client;
+
+import java.nio.charset.StandardCharsets;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+
+/**
+ * Unit test for {@link ImapClientRespReader}.
+ *
+ * @author kaituo
+ *
+ */
+public class ImapClientRespReaderTest {
```

Comment:
What about case for multiple literal items in a single response?

Reply:
Test was there already.  aka, testDecodeMultipleLiteralResponse() . (line 59)